contrib_bots/lib/translate.py

```diff
@@ -0,0 +1,86 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+from __future__ import print_function
+import sys
+import logging
+try:
+    from PyDictionary import PyDictionary as dictionary
+except ImportError:
+    logging.error("Dependency Missing!")
+    sys.exit(0)
+
+#Uses Python's Dictionary module
+#  pip install PyDictionary
+
+def get_translate_result(original_content):
+    if len(original_content.split()) < 2:
+        return "Not enough cases"
+    elif len(original_content.split()) > 3:
+        return "Too many cases"
+    else:
+        main_content = original_content.strip().split(' ', 1)[1]
+        if main_content == "help":
+            help_message = "To use this plugin start messages with @translate \
+                            followed by the word you want to translate and the \
+                            language code (all language codes can be found in \
+                            the Translate_Codes.md file found in the translate \
+                            folder). Example usage of this bot is \
+                            '@translate hello fr' or '@translate hello zh-TW'."
+            return help_message
+        else:
+            search_keyword = main_content.strip().split(' ', 1)[0]
+            country_code = main_content.strip().split(' ', 1)[1]
+            try:
+                message = dictionary.translate(search_keyword, search_keyword, country_code)
+            except Exception as e:
+                logging.exception(e)
+                return e
```

Comment:
You don't have to return the exception, logging is enough.
Or maybe you could reply the user with something like `Error in translation bot`